### PR TITLE
fix: [CDS-41790]: template height not visible

### PR DESCRIPTION
**Pull request by dhruvUpadhyay07**
#### Screenshots

NA

Screenshot 2022-08-12 at 10 38 59 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 1 Issues 
* Code issue 1 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10582)

File: src/modules/70-pipeline/components/PipelineStudio/StepCommands/StepCommands.tsx [hunk 1]
```diff
@@ -226,7 +226,7 @@ export function StepCommands(
           <Container>{getStepWidgetWithFormikRef()}</Container>
         </Layout.Vertical>
       ) : (
-        <div className={css.helpPanelGrid}>
+        <div className={props.helpPanelVisible ? css.helpPanelGrid : css.helpPanelGridNormal}>
```
Comment (yc147) [inline src/modules/70-pipeline/components/PipelineStudio/StepCommands/StepCommands.tsx:229]: If you are doing this

File: src/modules/70-pipeline/components/PipelineStudio/StepCommands/StepCommands.tsx [hunk 2]
```diff
@@ -226,7 +226,7 @@ export function StepCommands(
...
           <Container
             width={width}
```
Comment (yc147) [inline src/modules/70-pipeline/components/PipelineStudio/StepCommands/StepCommands.tsx:231]: then you don't need to set width here.

**Comment by dhruvUpadhyay07** [comment]
retrigger titlecheck

**Comment by dhruvUpadhyay07** [comment]
retrigger jest

**Comment by dhruvUpadhyay07** [comment]
retrigger jest

**Comment by dhruvUpadhyay07** [comment]
retrigger jest

File: src/modules/70-pipeline/components/PipelineStudio/StepCommands/StepCommands.module.scss [hunk 3]
```diff
@@ -10,10 +10,16 @@
...
-.helpPanelGrid {
+.helpPanelGridVisible {
   display: flex;
   direction: row;
   height: 0px;
+  width: 88%;
+}
+.helpPanelGridHidden {
+  display: flex;
+  direction: row;
```
Comment (yc147) [inline src/modules/70-pipeline/components/PipelineStudio/StepCommands/StepCommands.module.scss:21]: `direction` is an invalid prop. It should be `flex-direction`. And better use `Layout.Vertical` here.

**Review by yc147 — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**